__tests__/app.test.js

```diff
@@ -444,3 +444,29 @@ describe("DELETE /api/comments/comment_id", () => {
       })
   })
 })
+
+describe("GET /api/users", () => {
+  test("200: Responds with an array of all the users", () => {
+    return request(app) 
+    .get("/api/users")
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.users.length).toBeGreaterThan(1)
+      body.users.forEach((user) => {
+        expect(user).toMatchObject({
+          username: expect.any(String),
+          name: expect.any(String),
+          avatar_url: expect.any(String),
+        })
+      })
+    })
+  })
+  test("404: responds with 'Route not found' for invalid endpoint", () => {
+    return request(app)
+      .get("/api/user")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Route not found");
+      })
+  })
```

Comment:
This is also really good, but should not be in this describe block as it doesnt have anything to do with the endpoint `/api/users`. This test is a generic invalid endpoint and would the same `/api/banana`. This should also be in its own describe block